keras_cv/layers/patch_encoder.py

```diff
@@ -0,0 +1,21 @@
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+"""
+Linear projection of patches made with `Patching` layer
+"""
+#@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class PatchEncoder(layers.Layer):
+    def __init__(self, num_patches, projection_dim):
+        super(PatchEncoder, self).__init__()
```

Comment:
in python 3 you can rewrite `super(PatchEncoder, self).__init__()` to just `super().__init__()`

Reply:
Oh wow, I totally missed that for quite some time 🤦‍♂️

Reply:
Something we both learned from @fchollet!